charts/incubator/youtrack/questions.yaml

```diff
@@ -0,0 +1,141 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{controller}
+# Include{replicas}
+# Include{replica1}
+# Include{controllerExpertExtraArgs}
+# Include{containerConfig}
+# Include{serviceRoot}
+        - variable: main
+          label: "Main Service"
+          description: "The Primary service on which the healthcheck runs, often the webUI"
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+# Include{serviceSelectorLoadBalancer}
+# Include{serviceSelectorExtras}
+                    - variable: main
+                      label: "Main Service Port Configuration"
+                      schema:
+                        additional_attrs: true
+                        type: dict
+                        attrs:
+                          - variable: port
+                            label: "Port"
+                            description: "This port exposes the container port on the service"
+                            schema:
+                              type: int
+                              default: 8080
+                              required: true
+# Include{serviceSelectorLoadBalancer}
+# Include{serviceSelectorExtras}
+# Include{serviceExpertRoot}
+            default: false
+# Include{serviceExpert}
+# Include{serviceList}
+# Include{persistenceRoot}
+# Include{persistenceBasic}
+        - variable: data
+          label: "Database Storage"
+          description: "This is the location for the database (destination)."
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+# Include{persistenceBasic}
+        - variable: conf
+          label: "Configuration Storage"
+          description: "This is the location for the configuration (destination)."
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+# Include{persistenceBasic}
+        - variable: logs
+          label: "Logs Storage"
+          description: "This is the location for the logs (destination)."
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+# Include{persistenceBasic}
```

Comment:
Storage seems to be an issue with catalogue not like attrs. Written in line with other charts though so not sure what is going on? @stavros-k

Reply:
I'm wondering which chart did you use a base.
I don't recall any `values.yaml` or `Chart.yaml` to contain `---` on the top

Reply:
Not quite sure now I'll take them out. It might have been a remnant from my listing to get rid of the end of line errors. I scouted out a few to see how others had done it I'll remove the ---.

Reply:
*linting not listing...

Reply:
> Storage seems to be an issue with catalogue not like attrs. Written in line with other charts though so not sure what is going on? @stavros-k

persistenceBasic goes to the end, not the start of each variable block